src/guillotine/fetchContent.ts

```diff
@@ -1034,14 +1034,14 @@ function populateEnonicHeaders(context: Context) {
     context.req = req;
 }
 
-export async function fetchContentPathsForAllLocales(path: string, query: string = GET_STATIC_PATHS_QUERY, countPerLocale?: number): Promise<ContentPathItem[]> {
+export async function fetchContentPathsForAllLocales(path: string, query: string = GET_STATIC_PATHS_QUERY, countPerLocale = 999): Promise<ContentPathItem[]> {
```

Comment:
We might want to paginate the request for more then 1000 static sites. 
Or we could specify all paths that needs to be staticlly genereted (eks: '/site/blog/*', 'site/product/*')

Reply:
Good idea, let's take it in the next versions